exposed-tests/src/test/kotlin/org/jetbrains/exposed/sql/tests/shared/ddl/DatabaseMigrationTests.kt

```diff
@@ -769,33 +778,34 @@ class DatabaseMigrationTests : DatabaseTestsBase() {
 
     private fun expectedCreateSequenceStatement(sequenceName: String) =
         "CREATE SEQUENCE${" IF NOT EXISTS".takeIf { currentDialectTest.supportsIfNotExists } ?: ""} " +
-            "$sequenceName START WITH 1 MINVALUE 1 MAXVALUE 9223372036854775807"
+            "$sequenceName START WITH 1 MINVALUE 1 MAXVALUE ${currentDialectTest.sequenceMaxValue}"
 
     private fun expectedDropSequenceStatement(sequenceName: String) =
         "DROP SEQUENCE${" IF EXISTS".takeIf { currentDialectTest.supportsIfNotExists } ?: ""} $sequenceName"
 
-    private val sequence = Sequence(
-        name = "my_sequence",
-        startWith = 1,
-        minValue = 1,
-        maxValue = 9223372036854775807
-    )
+    private val sequenceName by lazy { "custom_sequence" }
 
-    private val sequenceName = "custom_sequence"
-
-    private val tableWithoutAutoIncrement = object : IdTable<Long>("test_table") {
-        override val id: Column<EntityID<Long>> = long("id").entityId()
+    private val tableWithoutAutoIncrement by lazy {
+        object : IdTable<Long>("test_table") {
+            override val id: Column<EntityID<Long>> = long("id").entityId()
+        }
     }
 
-    private val tableWithAutoIncrement = object : IdTable<Long>("test_table") {
-        override val id: Column<EntityID<Long>> = long("id").autoIncrement().entityId()
+    private val tableWithAutoIncrement by lazy {
+        object : IdTable<Long>("test_table") {
+            override val id: Column<EntityID<Long>> = long("id").autoIncrement().entityId()
+        }
     }
 
-    private val tableWithAutoIncrementCustomSequence = object : IdTable<Long>("test_table") {
-        override val id: Column<EntityID<Long>> = long("id").autoIncrement(sequence).entityId()
+    private val tableWithAutoIncrementCustomSequence by lazy {
+        object : IdTable<Long>("test_table") {
+            override val id: Column<EntityID<Long>> = long("id").autoIncrement(sequence).entityId()
+        }
     }
 
-    private val tableWithAutoIncrementSequenceName = object : IdTable<Long>("test_table") {
-        override val id: Column<EntityID<Long>> = long("id").autoIncrement(sequenceName).entityId()
+    private val tableWithAutoIncrementSequenceName by lazy {
+        object : IdTable<Long>("test_table") {
+            override val id: Column<EntityID<Long>> = long("id").autoIncrement(sequenceName).entityId()
+        }
```

Comment:
I changed these to lazy properties to be able to make the sequence variable in the test a late-initialized property without running into an error that it is not initalized yet.